src/Services/Database.ts

```diff
@@ -93,6 +94,8 @@ export class Database {
 
     private initModels(): void {
         Node.initialize(this.sequelize);
+        DiskInformation.initialize(this.sequelize);
+        Node.hasMany(DiskInformation);
```

Comment:
Why is this here? You can define this in the model, same as you did DiskInformation with `belongsTo`

Reply:
I must've left it by accident while testing something, but i agree it's not supposed to be here.